.github/workflows/main.yml

```
@@ -1,26 +1,40 @@
 
 jobs:
   deploy:
-    name: Delploy On CU Prod Server
+    name: Deploy to Heroku
     runs-on: ubuntu-latest
 
     steps:
-      - name: Check out code
-        uses: actions/checkout@v2      
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)